.husky/pre-commit

```diff
@@ -1,3 +1,5 @@
+eval "$(fnm env --use-on-cd)"
```

Comment:
not sure if this is the type of repo where this matters, but i'd generally put this in a separate commit/PR